docs/client-metrics.md

```diff
@@ -86,62 +86,73 @@ The event stream is a log of events and the time they occurred while the user is
 * error.&lt;image_url&gt;.profile.997 - a profile image could not load.
+* error.&lt;screen_name&gt;.&lt;service&gt;.998 - System unavailable.
 * loaded - logged after the first screen is rendered.
 * &lt;screen_name&gt;.submit - A submit event has occurred and all of the form's input elements are valid.
 * &lt;screen_name&gt;.refresh - The aforementioned screen was refreshed.
 
-### Events per screen
-
-
-#### account_unlock_complete
+#### Authentication namespace
+* error.&lt;screen_name&gt;.auth.114 - account has been throttled ("Attempt limit exceeded")
+* error.&lt;screen_name&gt;.auth.1005 - request taking longer than expected ("Working...")
+* error.&lt;screen_name&gt;.auth.999 - Unknown error.
 
-#### cannot_create_account
+### Events per screen
 
 #### change_password
 * error.change_password.auth.103 - incorrect password
 * error.change_password.auth.121 - account locked
+* error.change_password.auth.1008 - new password must be different
+* error.change_password.auth.1009 - password must be at least 8 characters
+* error.change_password.auth.1010 - valid password required
 * change_password.unlock-email.send - user attempted to send unlock email
 * change_password.unlock-email.send.success - unlock email successfully sent
 * change_password.success - password changed successfully
 
 #### choose_what_to_sync
 * choose-what-to-sync.engine-unchecked.`<engine_name>` - a Sync engine was unselected.
 
-#### complete_account_unlock
-* error.complete_account_unlock.auth.1025 - User clicked on an expired verification link.
-* error.complete_account_unlock.auth.1026 - User clicked on a damaged verification link.
-* complete_account_unlock.verification.success - successful verification
-
 #### complete_reset_password
 * complete_reset_password.verification.success - email successfully verified.
 * complete_reset_password.resend - A verification email was resent after an expired link was opened.
+* error.complete_reset_password.auth.1004 - password does not match
+* error.complete_reset_password.auth.1009 - password must be at least 8 characters
+* error.complete_reset_password.auth.1010 - valid password required
+* error.complete_reset_password.auth.1011 - valid email required
 * error.complete_reset_password.auth.1025 - User clicked on an expired verification link.
-* error.complete_reset_password.auth.1026 - User clicked on a damaged verification link.
 
-#### complete_sign_up
-* complete_sign_up.verification.success - email successfully verified.
-* complete_sign_up.resend - A verification email was resent after an expired link was opened.
-* error.complete_sign_up.auth.1025 - User clicked on an expired verification link.
-* error.complete_sign_up.auth.1026 - User clicked on a damaged verification link.
-* error.complete_sign_up.auth.1040 - User tried to verify an account that does not exist.
-* error.complete_sign_up.auth.1041 - User tried to verify a sign-in that has already been verified.
+#### verify_email
```

Comment:
@shane-tomlinson in datadog `complete_sign_up` does not appear, seems like it has been replaced with `verify_email`?